tests/scores/categorical/test_multicategorical.py

```diff
@@ -253,6 +253,30 @@ def test_firm(
     )
 
 
+def test_firm_dask():
+    """Tests firm works with dask"""
+    calculated = firm(
+        mtd.DA_FCST_FIRM.chunk(),
+        mtd.DA_OBS_FIRM.chunk(),
+        0.7,
+        [0, 5],
+        mtd.LIST_WEIGHTS_FIRM2,
+        0,
+        reduce_dims=None,
+        preserve_dims=["i", "j", "k"],
+    )
+
+    calculated = calculated.transpose(*["i", "j", "k"])
```

Comment:
The * is the unpack operator, so it would be clearer to say `calculated = calculated.transpose("i", "j", "k")`

Reply:
updated